grabdish/inventory-springboot/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM maven:3.6.3-openjdk-11 AS maven_build
 COPY pom.xml /tmp/
 COPY src /tmp/src/
 WORKDIR /tmp/
-RUN mvn package
+RUN mvn clean package spring-boot:repackage
```

Comment:
please move the build out into build.sh rather than in the dockerfile (see inventory-helidon for example)